clients/mercury.go

```diff
@@ -44,11 +44,25 @@ func (client *mercuryClient) NetworkParams() *chaincfg.Params {
 
 func (client *mercuryClient) GetUTXOs(ctx context.Context, address string, limit, confitmations int64) ([]UTXO, error) {
 	utxos := []UTXO{}
-	resp, err := http.Get(fmt.Sprintf("%s/utxo/%s?limit=%d&confirmations=%d", client.URL, address, limit, confitmations))
-	if err != nil || resp.StatusCode != http.StatusOK {
+	var resp *http.Response
+	var err error
+	for {
```

Comment:
I think it would be good to set a maximum retry time.

Reply:
From the perspective of the Darknodes, there’s no sane thing to do other than retry until it eventually succeeds.